ui/pages/confirm-add-suggested-nft/confirm-add-suggested-nft.js

```diff
@@ -174,12 +174,12 @@ const ConfirmAddSuggestedNFT = () => {
           display={Display.Flex}
           justifyContent={JustifyContent.center}
         >
-          <SiteOrigin
+          <TagUrl
             chip
-            siteOrigin={originMetadata.origin}
+            label={originMetadata.origin}
             title={originMetadata.origin}
-            iconSrc={originMetadata.iconUrl}
-            iconName={originMetadata.hostname}
+            src={originMetadata.iconUrl}
+            lockIconProps={{ name: originMetadata.hostname }}
```

Comment:
I don't think this is necessary? name refers to the name of the icon which is here: https://github.com/strawhatrag/metamask-extension/blob/b1279fec75844565036b96767633cd53dfe7262d/ui/components/component-library/icon/icon.types.ts#L28

I think this can be removed

```suggestion
```

Reply:
Removed Props in TagUrl component

test results 
![Screenshot 2024-05-07 194514](https://github.com/MetaMask/metamask-extension/assets/77497081/58fca2f7-f01b-4f64-8502-164fa46280bd)


Storybook
![chrome_0VfYDExTw6](https://github.com/MetaMask/metamask-extension/assets/77497081/5ff202bb-6948-49a7-bb22-d8599eb4be4d)